src/agent/onefuzz-task/src/tasks/coverage/dotnet.rs

```diff
@@ -56,6 +56,18 @@ impl Config {
     }
 }
 
+impl GetExpand for Config {
+    fn get_expand(&self) -> Result<Expand<'_>> {
+        Ok(self
+            .common
+            .get_expand()?
+            .target_exe(&self.target_exe)
```

Comment:
Do we need to expand target_exe if we're going to override it later here?

```rust
        let expand = self
            .config
            .get_expand()?
            .input_path(input)
            .target_exe(&target_exe); <---- here
```

Reply:
My intention for setting it in both places was to have a sort of insurance policy in case new code is ever added that initializes an `Expand` object and doesn't / forgets to set the `target_exe`.

You're right though, it's effectively dead code right now. I'd be very easily convinced to remove it.

Reply:
It's not the same `target_exe` though right? Setting it in `get_expand` is purely from the config but later, we populate it from `self.target_exe`

Reply:
The difference between the two is that the value gotten from `self.target_exe()` is the expanded version of what's in the config and it's guaranteed to exist in the filesystem.
It doesn't make sense to me to try and fit that validation into the config instantiation nor does it make sense to do it just in `get_expand()`. So, I think you're right, I'll remove it from `get_expand()` and just let it be initialized in that function.